api/src/main/java/org/pmiops/workbench/api/ProfileController.java

```diff
@@ -77,7 +77,6 @@
 
   @Override
   public ResponseEntity<CreateAccountResponse> createAccount(CreateAccountRequest request) {
-    // TODO(dmohs): Handle errors.
```

Comment:
FWIW, leaving the TODO in (with a link to a bug) is still a good idea.

Reply:
I thought about that, but it seems misleading—it would lead me to believe that once we have good error handling this is the only place it would need to be addressed, but actually we'd want to do a code audit.